aws/resource_aws_sns_topic_subscription.go

```diff
@@ -328,3 +328,18 @@ func obfuscateEndpoint(endpoint string) string {
 	}
 	return obfuscatedEndpoint
 }
+
+func snsSubscriptionAttributeUpdate(snsconn *sns.SNS, tfId string, tfName string, val string) error {
```

Comment:
This is very handy here. Can we update the signature of this function to just take in the expected values?

```go
func snsSubscriptionAttributeUpdate(snsconn *sns.SNS, subscriptionArn, attributeName, attributeValue string) error {
```

Thanks!

Reply:
No problem, done. This syntax is cleaner in fact